public/css/home.css

```
@@ -1256,3 +1256,25 @@ p#large_clock_period {
   opacity: 0;
   transition: opacity 300ms 0ms, visibility 0ms 300ms;
 }
+
+#updates-container {
+  margin-top: 2em;
+}
+.info-header a {
+  font-size: 120%;
+  color: var(--green2);
+  text-decoration: none;
+  margin-bottom: 2em;
+}
+
+
+hr {
+  margin-top:0px;
+}
+#updates ul {
+  margin-bottom: 20px;
+}
+#moreVersions {
```

Use kebab-case for CSS IDs and classes, not camelCase
```suggestion
#more-versions {
```